Sources/Function.swift

```diff
@@ -3,25 +3,38 @@
 func elementString<T>(_ x: T, debug: Bool, pretty: Bool) -> String {
     let mirror = Mirror(reflecting: x)
 
-    let typeName = type(of: x)
+    let typeName = String(describing: mirror.subjectType)
 
+    // Empty
     if mirror.children.count == 0 {
         return valueString(x, debug: debug)
-    } else {
-        let prefix = "\(typeName)("
-        let fields = mirror.children.map {
-            "\($0.label ?? "-"): " + valueString($0.value, debug: debug)
-        }
+    }
 
-        if pretty, fields.count > 1 {
-            let tailFields = fields.dropFirst()
-                .map { $0.indent(size: prefix.count) }
-                .joined(separator: ",\n")
+    // Optional<T>
+    if typeName.starts(with: "Optional<") { // TODO: better judge is exist?
```

Comment:
As you've mentioned in slack, we can use  `displayStyle` property of mirror type.
https://developer.apple.com/documentation/swift/mirror/displaystyle

Reply:
The suggestion is not confirmed yet.

Reply:
Thanks for comment, I'll change it in another PR :-)